index.js

```diff
@@ -0,0 +1,330 @@
+load();
+
+function load() {
+  alert("Get ready to play the coolest game you've ever played!")
+}
+
+// assigning variables
+let playerOne = '';
+let playerTwo = '';
+let clockCount = 0;
+let interval;
+let timer = 0;
+
+// initializing variables with DOM scripting
+let startBtn = document.getElementById('start');
+startBtn.disabled = true;
+let onePlayer = document.getElementById('onePlayer');
+let twoPlayer = document.getElementById('twoPlayer');
+let playerOneName = document.getElementById('playerOne')
+let playerTwoName = document.getElementById('playerTwo')
+let currentPlayer = 'x';
+let status = document.getElementById('status-bar');
+let clock = document.getElementById('clock');
+let cells = Array.from(document.getElementsByTagName('td'));
+let cellOne = document.getElementById('cell-1')
+let cellTwo = document.getElementById('cell-2')
+let cellThree = document.getElementById('cell-3')
+let cellFour = document.getElementById('cell-4')
+let cellFive = document.getElementById('cell-5')
+let cellSix = document.getElementById('cell-6')
+let cellSeven = document.getElementById('cell-7')
+let cellEight = document.getElementById('cell-8')
+let cellNine = document.getElementById('cell-9')
+let usedCellArray = [];
+let gameMode = '';
+
+function capitalize(string) {
+    string = string[0].toUpperCase() + string.slice(1).toLowerCase();
+    return string;
+}
+
+function initialize() {
+
+}
+
+twoPlayer.addEventListener('mouseover', (event) => {
+  event.target.style.color = '#42aff7'
+})
+
+twoPlayer.addEventListener('mouseout', (event) => {
+  event.target.style.color = 'black'
+})
+
+onePlayer.addEventListener('mouseover', (event) => {
+  event.target.style.color = '#42aff7'
+})
+
+onePlayer.addEventListener('mouseout', (event) => {
+  event.target.style.color = 'black'
+})
+
+startBtn.addEventListener('mouseover', (event) => {
+  event.target.style.color = '#42aff7'
+})
+
+startBtn.addEventListener('mouseout', (event) => {
+  event.target.style.color = '42aff7'
+})
```

Comment:
All of this could be done through CSS, and could be done just by adding a class to all of the buttons, then adding a hover effect on that class as well. It would cut the code by more then half